CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `ignore_index` to to retrieval metrics ([#676](https://github.com/PyTorchLightning/metrics/pull/676))
 
 
+- Added support for multi references in `ROUGEScore` ([#680](https://github.com/PyTorchLightning/metrics/pull/680))
+
+
```

Comment:
@stancld Had the same question. Although since @SkafteNicki made the commit, I thought that would be the correct way. I can move it to `Changed` if need be.

Reply:
So i put it in `added` since we do remove the old feature (single reference evaluation) but just "add on top" of it. It really does not matter that much :]